requirements-doc.txt

```diff
@@ -3,7 +3,7 @@
 # same reason, we force devito==4.8.7 as later versions of devito
 # require numpy>=2.0.0
 numpy>=1.21.0,<2.0.0
-scipy>=1.11.0
+scipy>=1.11.0,<1.13
```

Comment:
I guess this is for pytensor/pymc to work?

Reply:
This is for arviz to work:

https://discourse.pymc.io/t/importerror-cannot-import-name-gaussian-from-scipy-signal/14170

If we could get arviz to 0.18 we wouldn't need it, but other package dependencies prevent that from happening. It's a mess!